src/spring/azext_spring/_deployment_factory.py

```diff
@@ -238,20 +250,38 @@ def deployment_settings_options_from_resource(original):
         'instance_count': original.sku.capacity,
         'sku': original.sku,
         'env': original.properties.deployment_settings.environment_variables,
-        'config_file_patterns': _get_origin_config_file_patterns(original.properties.deployment_settings.addon_configs)
+        'config_file_patterns': _get_origin_config_file_patterns(original.properties.deployment_settings.addon_configs),
+        'custom_actuator_port': _get_origin_custom_actuator_port(original.properties.deployment_settings.addon_configs),
+        'custom_actuator_path': _get_origin_custom_actuator_path(original.properties.deployment_settings.addon_configs)
     }
     if original.properties.deployment_settings.container_probe_settings is not None:
         options['disable_probe'] = original.properties.deployment_settings.container_probe_settings.disable_probe
     return options
 
 
 def _get_origin_config_file_patterns(origin_addon_configs):
-    if origin_addon_configs:
+    if APPLICATION_CONFIGURATION_SERVICE_NAME in origin_addon_configs:
```

Comment:
Will this work if the origin_addon_configs is None?

Reply:
not work, it only checks the values from origin app deployment